src/components/shared/helpers/HelperUtils.test.ts

```diff
@@ -0,0 +1,296 @@
+      'Disaster Type': 'Earthquakes',
+      Region: 'Asia',
+      Subregion: 'South-eastern Asia',
+      'Country of Implementation': 'Indonesia',
+      'Supporting Partners':
+        'Digicel, International Organisation for Migration (IOM)',
+      'Un Host Organisation': 'UN Pulse Lab Jakarta',
+      Data: 'Mobile Network Data, Spatial Data',
+      SDG: 'SDG 9',
+      'Date of Implementation': 2019,
+      Source:
+        'https://www.unglobalpulse.org/2019/12/understanding-population-movement-after-the-2018-central-sulawesi-natural-disasters/',
+      imgurl:
+        'https://www.unglobalpulse.org/wp-content/uploads/2019/12/PLJ-Palu-visualization.webp'
+    }
+  ];
+
+  const parameterFilters = {
+    Country: [],
+    Data: [],
+    'Disaster Type': [],
+    Region: [],
+    SDG: [],
+    'Sub Region': [],
+    'UN Host': []
+  };
+
+  it('should return an empty list when there are no params set', () => {
+    const results = getParameterFilteredProjects(
+      parameterFilters,
+      testProjects,
+      0
+    );
+
+    expect(results).toHaveLength(0);
+  });
+
+  it('should filter by region', () => {
+    const results = getParameterFilteredProjects(
+      {
+        ...parameterFilters,
+        Region: [{ label: 'Asia', value: 'asia' }]
+      },
+      testProjects,
+      1
+    );
+
+    expect(results.length).toBeGreaterThan(0);
+    results.map((r: any) => expect(r.Region).toBe('Asia'));
+  });
+
+  it('should filter by subregion parameter', () => {
+    const results = getParameterFilteredProjects(
+      {
+        ...parameterFilters,
+        'Sub Region': [
+          {
+            label: 'South-eastern Asia',
+            value: 'south-eastern asia'
+          }
+        ]
+      },
+      testProjects,
+      1
+    );
+
+    expect(results.length).toBeGreaterThan(0);
+    results.map((r: any) =>
+      expect(r.Subregion.includes('South-eastern Asia')).toBeTruthy()
+    );
+  });
+
+  it('should filter by country parameter', () => {
+    const results = getParameterFilteredProjects(
+      {
+        ...parameterFilters,
+        Country: [
+          {
+            label: 'Indonesia',
+            value: 'indonesia'
+          }
+        ]
+      },
+      testProjects,
+      1
+    );
+
+    expect(results.length).toBeGreaterThan(0);
+    results.map((r: any) =>
+      expect(r['Country of Implementation'].includes('Indonesia')).toBeTruthy()
+    );
+  });
+
+  it('should filter by disaster type parameter', () => {
+    const results = getParameterFilteredProjects(
+      {
+        ...parameterFilters,
+        'Disaster Type': [
+          {
+            label: 'Earthquakes',
+            value: 'earthquakes'
+          }
+        ]
+      },
+      testProjects,
+      1
+    );
+
+    expect(results.length).toBeGreaterThan(0);
+    results.map((r: any) =>
+      expect(r['Disaster Type'].includes('Earthquakes')).toBeTruthy()
```

Comment:
On second thought I think it's better to do `expect(r['Disaster Type]).toContain('Earthquakes'))` here because it gives you better output if it fails - I'll do a pass of these and change them all tomorrow